packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -50,8 +50,15 @@ export class BlobHandle implements IFluidHandle<ArrayBufferLike> {
 
 export class BlobManager {
     public static readonly basePath = "_blobs";
-    private readonly pendingBlobIds: Map<string, Deferred<void>> = new Map();
+    // uploaded blob IDs
     private readonly blobIds: Set<string> = new Set();
+    // blobs for which upload is pending. maps to a promise that will resolve once the blob has been uploaded and a
+    // BlobAttach op has round-tripped.
+    private readonly pendingBlobIds: Map<string, Deferred<void>> = new Map();
+    // blobs uploaded while detached; cleared upon attach
+    private readonly detachedBlobIds: Set<string> = new Set();
+    // map of detached blob IDs to IDs used by storage. used to support blob handles given out while detached
+    private redirectTable: Map<string, string> | undefined;
```

Comment:
I do not see redirectTable being summarized. I believe this makes it impossible for client loading from snapshot to resolve blob handles. Is this covered in UTs?

Reply:
This is not implemented in this PR. Loading from snapshot (after creating empty file) is not working on ODSP and other drivers don't support empty file creation.

Reply:
Still true, or did avoiding the file rename unblock this work?

Reply:
I meant in the first real summary that we produce as part of this flow. redirectTable has to be serialized in summary in some form or shape to be able to map from internal IDs (coming from detached storage) to real storage IDs.

Reply:
And I do see that you serialize it below, I think it was not the case when I left this comment